src/lib/ibcclient.ts

```diff
@@ -171,35 +179,32 @@ function createBroadcastTxErrorMessage(result: BroadcastTxFailure): string {
   return `Error when broadcasting tx ${result.transactionHash} at height ${result.height}. Code: ${result.code}; Raw log: ${result.rawLog}`;
 }
 
-// TODO: replace this with buildFees in IbcClient constructor to take custom gasPrice, override gas needed
-const fees = {
-  initClient: {
-    amount: coins(2500, 'ucosm'),
-    gas: '100000',
-  },
-  updateClient: {
-    amount: coins(10000, 'ucosm'),
-    gas: '400000',
-  },
-  initConnection: {
-    amount: coins(2500, 'ucosm'),
-    gas: '100000',
-  },
-  connectionHandshake: {
-    amount: coins(5000, 'ucosm'),
-    gas: '200000',
-  },
-  initChannel: {
-    amount: coins(2500, 'ucosm'),
-    gas: '100000',
-  },
-  channelHandshake: {
-    amount: coins(5000, 'ucosm'),
-    gas: '200000',
-  },
+export interface IbcFeeTable extends FeeTable {
+  readonly initClient: StdFee;
+  readonly updateClient: StdFee;
+  readonly initConnection: StdFee;
+  readonly connectionHandshake: StdFee;
+  readonly initChannel: StdFee;
+  readonly channelHandshake: StdFee;
+}
+
+export type IbcClientOptions = SigningStargateClientOptions & {
+  gasLimits?: Partial<GasLimits<IbcFeeTable>>;
+};
+
+const defaultGasPrice = GasPrice.fromString('0.025ucosm');
+const defaultGasLimits: GasLimits<IbcFeeTable & CosmosFeeTable> = {
+  send: 80000,
+  initClient: 100000,
+  updateClient: 400000,
+  initConnection: 100000,
+  connectionHandshake: 200000,
+  initChannel: 100000,
+  channelHandshake: 100000,
```

Comment:
That should be 200000.

I did some minor tune-ing of prices to get the values on the left. I think they are all within a factor of 2 (you cannot safely half them). updateClient can likely go down a bunch in tests, but if there are 100 validators, then not.